src/actions/collateral.ts

```diff
@@ -1,85 +1,87 @@
-import { Constructor, TinlakeParams } from '../Tinlake'
-import { waitAndReturnEvents, executeAndRetry } from '../services/ethereum'
+import { Constructor, TinlakeParams, PendingTransaction } from '../Tinlake'
 import BN from 'bn.js'
 
 export function CollateralActions<ActionsBase extends Constructor<TinlakeParams>>(Base: ActionsBase) {
   return class extends Base implements ICollateralActions {
     mintTitleNFT = async (nftAddr: string, user: string) => {
-      const nft: any = this.eth.contract(this.contractAbis['COLLATERAL_NFT']).at(nftAddr)
-      const txHash = await executeAndRetry(nft.issue, [user, this.ethConfig])
-      console.log(`[Mint NFT] txHash: ${txHash}`)
-      const res: any = await waitAndReturnEvents(
-        this.eth,
-        txHash,
-        this.contractAbis['COLLATERAL_NFT'],
-        this.transactionTimeout
-      )
-      return res.events[0].data[2].toString()
+      // TODO: this is untested right now
```

Comment:
Is this still open?

Reply:
Yes, but this function is never used (except in a non-functioning test), so I'd suggest we fix that later.

Reply:
If it is not needed, can we comment it out and add the TODO to re-add if needed?